src/dispensary.js

```diff
@@ -261,7 +263,11 @@ export default class Dispensary {
         hashes.add(hash);
       }
 
-      resolve(Array.from(hashes));
+      var hashesArray = Array.from(hashes).sort((a, b) => {
+        return natCompare(a.match(/\s+(.*)/)[1], b.match(/\s+(.*)/)[1]);
```

Comment:
Do our filename results ever have whitespace? Why the odd regex instead of just compared `a` and `b`?